src/components/Builder/Achievement/index.jsx

```diff
@@ -20,39 +23,45 @@ import {
 
 const Achievement = ({ achievementData }) => {
   Achievement.propTypes = {
-    achievementData: PropTypes.object
+    achievementData: PropTypes.object.isRequired,
   };
+
   const { profile_id } = useParams();
   const [createAchievementService] = useCreateAchievementMutation();
+  const [updateAchievementService] = useUpdateAchievementMutation();
   const [form] = Form.useForm();
   const [activeKey, setActiveKey] = useState("0");
   const [items, setItems] = useState([
-    { label: "Achievement 1", children: null, key: "0" }
+    {
+      label: "Achievement 1",
+      children: null,
+      key: "0",
+      isExisting: "",
```

Comment:
default value should be false for `isExisting`

Reply:
Done!